nats-native/src/main/java/org/ballerinalang/nats/Constants.java

```diff
@@ -128,9 +125,11 @@
     public static final BString MAX_BYTES = StringUtils.fromString("maxBytes");
 
     // Strand meta data
-    public static final StrandMetadata ON_MESSAGE_METADATA = new StrandMetadata(ORG_NAME, NATS, VERSION,
+    public static final StrandMetadata ON_MESSAGE_METADATA = new StrandMetadata(ORG_NAME, NATS,
```

Comment:
We can create the StrandMetadata when we need it, and use the `getModule().getOrg()`, `getModule().getName()`, etc. to handle these.

Reply:
Fixed in c8e94613ed7ff23ec27267c9e3a2bb186a1d285e